app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from shutil import copyfileobj
+
+
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3:
+        return
+
+    cp, source_file, destination_file = parts
+    if source_file == destination_file:
```

Comment:
Write one if statement that will abandon the execution, there is no need to split logic in two or more if conditions here.